alkos/kernel/src/panic.cpp

```diff
@@ -1,13 +1,12 @@
 /* internal includes */
+#include <arch_utils.hpp>
 #include <panic.hpp>
 #include <terminal.hpp>
 
-extern "C" void KernelPanic(const char *msg) {
+extern "C" void KernelPanic(const char *msg)
+{
     TerminalWriteError("[ KERNEL PANIC ]\n");
     TerminalWriteError(msg);
 
-    asm volatile("cli");
```

Comment:
Another thing that needs to be provided by the abi. Inavoidable, yes - that's why i think we need to keep track of those things, like have a list of  functions needed to be provided by the arch in some file or list. And keep it minimal

Reply:
Will be addressed on some sync call.